influence_benchmark/experiments/slurm/expert_iteration.sh

```diff
@@ -10,7 +10,8 @@ export CONFIG_NAME="EI_test_up"
 # SLURM job parameters
 export SLURM_CPUS_PER_TASK=128
 export SLURM_MEM="300gb"
-export SLURM_GPUS="A6000:8"
+export SLURM_GPUS="8"
+export NODE_LIST="ddpg.ist.berkeley.edu,dqn.ist.berkeley.edu,gail.ist.berkeley.edu,gan.ist.berkeley.edu"
```

Comment:
I'm confused as to why this and related deltas appears in this PR?

Reply:
I think you merged with main but never dev?

Reply:
I tried doing a merge from main to dev, but somehow didn't seem to show up?